docs/framework/migration-guide/toc.yml

```diff
@@ -1,242 +1,242 @@
+      - name: 4.5 to 4.7
+        href: retargeting/4.5-4.7.md
+      - name: 4.5 to 4.7.1
+        href: retargeting/4.5-4.7.1.md
+      - name: 4.5 to 4.7.2
+        href: retargeting/4.5-4.7.2.md
+      - name: 4.5 to 4.8
+        href: retargeting/4.5-4.8.md
+      - name: 4.5.1 to 4.5.2
+        href: retargeting/4.5.1-4.5.2.md
+      - name: 4.5.1 to 4.6
+        href: retargeting/4.5.1-4.6.md
+      - name: 4.5.1 to 4.6.1
+        href: retargeting/4.5.1-4.6.1.md
+      - name: 4.5.1 to 4.6.2
+        href: retargeting/4.5.1-4.6.2.md
+      - name: 4.5.1 to 4.7
+        href: retargeting/4.5.1-4.7.md
+      - name: 4.5.1 to 4.7.1
+        href: retargeting/4.5.1-4.7.1.md
+      - name: 4.5.1 to 4.7.2
+        href: retargeting/4.5.1-4.7.2.md
+      - name: 4.5.2 to 4.6
+        href: retargeting/4.5.2-4.6.md
+      - name: 4.5.2 to 4.6.1
+        href: retargeting/4.5.2-4.6.1.md
+      - name: 4.5.2 to 4.6.2
+        href: retargeting/4.5.2-4.6.2.md
+      - name: 4.5.2 to 4.7
+        href: retargeting/4.5.2-4.7.md
+      - name: 4.5.2 to 4.7.1
+        href: retargeting/4.5.2-4.7.1.md
+      - name: 4.5.2 to 4.7.2
+        href: retargeting/4.5.2-4.7.2.md
+      - name: 4.5.2 to 4.8
+        href: retargeting/4.5.2-4.8.md
+      - name: 4.6 to 4.6.1
+        href: retargeting/4.6-4.6.1.md
+      - name: 4.6 to 4.6.2
+        href: retargeting/4.6-4.6.2.md
+      - name: 4.6 to 4.7
+        href: retargeting/4.6-4.7.md
+      - name: 4.6 to 4.7.1
+        href: retargeting/4.6-4.7.1.md
+      - name: 4.6 to 4.7.2
+        href: retargeting/4.6-4.7.2.md
+      - name: 4.6 to 4.8
+        href: retargeting/4.6-4.8.md
+      - name: 4.6.1 to 4.6.2
+        href: retargeting/4.6.1-4.6.2.md
+      - name: 4.6.1 to 4.7
+        href: retargeting/4.6.1-4.7.md
+      - name: 4.6.1 to 4.7.1
+        href: retargeting/4.6.1-4.7.1.md
+      - name: 4.6.1 to 4.7.2
+        href: retargeting/4.6.1-4.7.2.md
+      - name: 4.6.1 to 4.8
+        href: retargeting/4.6.1-4.8.md
+      - name: 4.6.2 to 4.7
+        href: retargeting/4.6.2-4.7.md
+      - name: 4.6.2 to 4.7.1
+        href: retargeting/4.6.2-4.7.1.md
+      - name: 4.6.2 to 4.7.2
+        href: retargeting/4.6.2-4.7.2.md
+      - name: 4.6.2 to 4.8
+        href: retargeting/4.6.2-4.8.md
+      - name: 4.7 to 4.7.1
+        href: retargeting/4.7-4.7.1.md
+      - name: 4.7 to 4.7.2
+        href: retargeting/4.7-4.7.2.md
+      - name: 4.7 to 4.8
+        href: retargeting/4.7-4.8.md
+      - name: 4.7.1 to 4.7.2
+        href: retargeting/4.7.1-4.7.2.md
+      - name: 4.7.1 to 4.8
+        href: retargeting/4.7.1-4.8.md
+      - name: 4.7.2 to 4.8
+        href: retargeting/4.7.2-4.8.md
+  - name: Version compatibility
+    href: version-compatibility.md
+- name: .NET Framework versions
   items:
   - name: Versions and dependencies
     href: versions-and-dependencies.md
-  - name: Determine which .NET Framework versions are installed
+  - name: Installed versions
```

Comment:
Could be "Find installed versions" to more clearly represent the content while still shortening the text.